indexer/ftypes_matcher.hpp

```diff
@@ -282,7 +282,7 @@ class IsCoastlineChecker : public BaseChecker
 class IsHotelChecker : public BaseChecker
 {
 public:
-  enum class Type
+  enum class Type: unsigned
```

Comment:
Why?

Reply:
this a compile warning fix:
comparison of integer expressions of different signedness: ‘std::underlying_type_t<ftypes::IsHotelChecker::Type>’ {aka ‘int’} and ‘long unsigned int’